src/vlt/tap-snapshots/test/commands/help.ts.test.cjs

```diff
@@ -151,6 +151,18 @@ More documentation available at <https://docs.vlt.sh>
 
   --fetch-retry-maxtimeout=<n>
                        Maximum number of milliseconds between two retries
+  -i<name> --identity=<name>
```

Comment:
i cant imagine this is associated with your changes, but why is there not a newline before this?

Reply:
That's just jackspeak's behavior. It only adds extra newlines when the descriptions wrap. Otherwise, it starts to feel excessively spaced-out.